### ISSUE TITLE
NUnit item templates should have version in their identity

### ISSUE THREAD

**Issue by nohwnd** [dotnet/test-templates#265]
NUnit item templates should have version in their identity and get priority, just like the other templates.

**Comment by OsirisTerje** [comment]
What happens here?

**Comment by nohwnd** [comment]
This is mstest, we set version 7.0 in the template identity so it is different from mstest.6.0, and we set the precedence (priority) to 1000 more than the previous version, so when you install net6 and net7 sdks the templates won't clash and will be correctly ordered: 

https://github.com/dotnet/test-templates/blob/main/template_feed/Microsoft.DotNet.Test.ProjectTemplates.7.0/content/MSTest-CSharp/.template.config/template.json#LL9C10-L10C3

Then there are 2 template types for NUnit, project template (adds project) and item template (adds .cs file, for example). 

https://github.com/dotnet/test-templates/blob/main/template_feed/Microsoft.DotNet.Test.ProjectTemplates.7.0/content/NUnit-CSharp/.template.config/template.json#L9-L10

https://github.com/dotnet/test-templates/blob/main/template_feed/Microsoft.DotNet.Test.ProjectTemplates.7.0/content/NUnit-CSharp-Item/.template.config/template.json#L9

Neither of them are specifying version in the name, so whatever sdk you install as the last you will get it overwriting the other templates. And the item template does not even have precedence.
